kubernetes/RemoteResourceS3/resource.yaml

```diff
@@ -74,16 +74,16 @@ items:
     kind: CustomResourceDefinition
     metadata:
       # name must match the spec fields below, and be in the form: <plural>.<group>
-      name: remoteresourcess3.deploy.razee.io
+      name: remoteresourcess3."{{GROUP}}"
       annotations:
         razee.io/git-repo: "{{{GIT_REMOTE}}}"
         razee.io/commit-sha: "{{TRAVIS_COMMIT}}"
     spec:
       # group name to use for REST API: /apis/<group>/<version>
-      group: deploy.razee.io
+      group: "{{GROUP}}"
       # list of versions supported by this CustomResourceDefinition
       versions:
-        - name: v1alpha1
+        - name: "{{VERSION}}"
```

Comment:
you are overwriting `v1alpha1`, but you should be overwriting `v1alpha2` line 147.